tools/deviceGen/src/device_features.json

```diff
@@ -0,0 +1,2821 @@
+{
+    "vulkan10core": [
+        {
```

Comment:
the input to a tool should stay in `src/nbl/video/`, this structure would make someone think that this json is part of the tool's source which it is not

Reply:
I would not put the jsons into Nabla’s source directory rather rename the tool’s directory from “src” to “input” or similar since indeed it’s not the tools src but neither Nabla just an input for the tool

if it's really desired what I don't like then `NBL_ROOT_PATH` must be used to reference the inputs

Reply:
I changed the directory name from `src` to `input`

Reply:
its still a bit weird that the JSONs live under `tools/deviceGen` but oh well.

Reply:
> I would not put the jsons into Nabla’s source directory rather rename the tool’s directory from “src” to “input” or similar since indeed it’s not the tools src but neither Nabla just an input for the tool

I never meant for anything else other than the JSONs to be moved out of `tools/deviceGen`

Reply:
I'd really want to move `deviceGen/input` out of `tools` and into `src/nbl/video/device_capabilities` or somewhere else in the `src/nbl/video` directory

P.S. does the `output` usually get send to build dir?

Reply:
`output` doesn't get sent to build. It exists purely for the PR

Reply:
I have mostly been copying and pasting the gen'd files into `output` when I wanted to update it

Reply:
> I'd really want to move deviceGen/input out of tools and into src/nbl/video/device_capabilities or somewhere else in the src/nbl/video directory

@AnastaZIuk fix when merging